api/migrations/019_create_brokerlog_table.js

```diff
@@ -27,6 +27,8 @@ function up(knex) {
     table.string('id').primary();
 
     table.string('userId');
+    table.string('userFirstname');
+    table.string('userLastname');
```

Comment:
You can't modify existing migration file.

You should add a migration sctip to add thos column.